src/tests/ftest/util/dfuse_utils.py

```diff
@@ -84,14 +86,15 @@ def set_dfuse_cont_param(self, cont, display=True):
 class Dfuse(DfuseCommand):
     """Class defining an object of type DfuseCommand"""
 
-    def __init__(self, hosts, tmp, dfuse_env=False):
+    def __init__(self, hosts, tmp, log_file=None, dfuse_env=False):
         """Create a dfuse object"""
         super(Dfuse, self).__init__("/run/dfuse/*", "dfuse")
 
         # set params
         self.hosts = hosts
         self.tmp = tmp
         self.dfuse_env = dfuse_env
+        self.log_file
```

Comment:
(pylint-pointless-statement) Statement seems to have no effect
(pylint-no-member) Instance of 'Dfuse' has no 'log_file' member